spring-native-image/pom.xml_buildpacks

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think this is needed. Please drop the file.

Reply:
Followed in https://github.com/graalvm/graalvm-demos/pull/168/commits/5ad973dc312a856a73b47064964b9550ddfdf442

Reply:
I said file, not line :wink: